megamek/src/megamek/codeUtilities/MathUtility.java

```diff
@@ -18,75 +18,81 @@
-     * @param f   location factor between the two points
-     * @return float graphical linear interpolation value between min and max a f of
-     *         0f will return the minimum, a f of 1f will return the maximum.
-     *         Otherwise, this will return the float value between the two points
+     * @param min    the minimum value
+     * @param max    the maximum value
+     * @param factor location factor between the two points
+     *
+     * @return float graphical linear interpolation value between min and max. A factor of 0f will return the minimum, a
+     *       factor of 1f will return the maximum. Otherwise, this will return the float value between the two points
      */
-    public static float lerp(final float min, final float max, final float f) {
-        return min * (1f - f) + max * f;
+    public static float lerp(final float min, final float max, final float factor) {
+        return min * (1f - factor) + max * factor;
     }
 
     /**
-     * @param min the minimum value
-     * @param max the maximum value
-     * @param f   location factor between the two points
-     * @return long rounded graphical linear interpolation value between min and max
-     *         a f of 0d will return the minimum, a f of 1d will return the maximum.
-     *         Otherwise, this will return the rounded long between the two points
+     * @param min    the minimum value
+     * @param max    the maximum value
+     * @param factor location factor between the two points
+     *
+     * @return long rounded graphical linear interpolation value between min and max. A factor of 0d will return the
+     *       minimum, a factor of 1d will return the maximum. Otherwise, this will return the rounded long between the
+     *       two points
      */
-    public static long lerp(final long min, final long max, final double f) {
+    public static long lerp(final long min, final long max, final double factor) {
         // The order of operations is important here, to not lose precision
-        return Math.round(min * (1L - f) + max * f);
+        return Math.round(min * (1L - factor) + max * factor);
     }
     // endregion Linear Interpolation
 
     // region Clamp
+
     /**
      * @param value the int value to clamp
      * @param min   the minimum limit
      * @param max   the maximum limit
-     * @return The value if it is inside the range given by the limits (inclusive);
-     *         the min value if value is below that range and the max value if value
-     *         is above that range.
-     *         clamp(2, 6, 8) returns 6,
-     *         clamp(7, 6, 8) returns 7,
-     *         clamp(12, 3, 5) returns 5.
+     *
+     * @return The value if it is inside the range given by the limits (inclusive); the min value if value is below that
+     *       range and the max value if value is above that range. clamp(2, 6, 8) returns 6, clamp(7, 6, 8) returns 7,
+     *       clamp(12, 3, 5) returns 5.
```

Comment:
I'd argue that the list format here was A) deliberate, and B) clearer.

Reply:
Turned it into a list.